### PR TITLE
remove remaining Vault enterprise code from UI

### PR DESCRIPTION
**Pull request by JanMa**
Closes: #129

## Target Release

2.2.0

### CODE REVIEW

File: ui/tests/integration/helpers/changelog-url-for-test.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ module('Integration | Helper | changelog-url-for', function (hooks) {
   });
 
   test('it builds the url for double-digit versions', function (assert) {
-    const result = changelogUrlFor(['1.13.0+ent']);
+    const result = changelogUrlFor(['1.13.0']);
     assert.strictEqual(result, CHANGELOG_URL.concat('1130'));
```
Comment (cipherboy) [inline ui/tests/integration/helpers/changelog-url-for-test.js:31]: Don't we use dashes for ours? https://openbao.org/docs/release-notes/2-0-0/

File: ui/lib/core/addon/helpers/changelog-url-for.js [hunk 2]
```diff
@@ -22,8 +22,8 @@ export function changelogUrlFor([version]) {
   const url = 'https://www.github.com/openbao/openbao/blob/main/CHANGELOG.md#';
   if (!version) return url;
   try {
-    // strip the '+prem' from enterprise versions and remove periods
```
Comment (cipherboy) [inline ui/lib/core/addon/helpers/changelog-url-for.js:25]: I think we should continue to strip `+` suffixes. I've been setting `+gitlab.<build>` in my internal builds:

https://gitlab.com/gitlab-org/govern/secrets-management/openbao-internal/-/blob/main/Makefile?ref_type=heads#L8

File: ui/app/templates/vault/cluster/secrets/backend/error.hbs [hunk 3]
```diff
@@ -33,33 +33,19 @@
         and navigating from there.
       </p>
     {{else if (eq this.model.httpStatus 403)}}
-      {{#if (eq this.model.policyPath "sys/capabilities-self")}}
-        <p>
-          Your auth token does not have access to
-          <code>{{this.model.policyPath}}</code>. Vault Enterprise uses this endpoint to determine what actions are allowed
```
Comment (cipherboy) [inline ui/app/templates/vault/cluster/secrets/backend/error.hbs:39]: Is this necessarily true? I think we have this endpoint too: https://openbao.org/api-docs/system/capabilities-self/

File: ui/app/templates/vault/cluster/error.hbs [hunk 4]
```diff
@@ -13,27 +13,13 @@
     </p.levelLeft>
   </PageHeader>
   <div class="box is-sideless has-background-white-bis has-text-grey has-text-centered">
-    {{#if (and (eq this.model.httpStatus 403) (eq this.model.policyPath "sys/capabilities-self"))}}
```
Comment (cipherboy) [inline ui/app/templates/vault/cluster/error.hbs:16]: As above.

**Review by cipherboy — APPROVED** [review]
@JanMa I think some of the removals might be a little heavy (w.r.t. capabilities/self), but I've not checked to see if they truly are unused or not.

File: ui/tests/integration/helpers/changelog-url-for-test.js (hunk 1 above)
Comment (JanMa) [reply ui/tests/integration/helpers/changelog-url-for-test.js:31]: We did in the Alpha/Beta versions :thinking: Not sure yet if I want to test it's working properly for those versions

Comment (JanMa) [reply ui/tests/integration/helpers/changelog-url-for-test.js:31]: I'll leave it as is for now

**Pull request merged**